src/domain/version.ts

```diff
@@ -88,6 +88,6 @@ class Identifier {
 
   public constructor(value: string) {
     const numeric = parseInt(value);
-    this.value = isNaN(numeric) ? value : numeric;
+    this.value = /^\d+$/.test(value) ? numeric : value;
```

Comment:
This test for a numeric value was evaluating to true when the string began with a digit, e.g., `0rc1`.